src/dotnet/commands/dotnet-help/LocalizableStrings.resx

```diff
@@ -265,7 +265,7 @@
     <value>Version of the installed Shared Framework to use to run the application.</value>
   </data>
   <data name="RollForwardOnNoCandidateFxDefinition" xml:space="preserve">
-    <value>Roll forward on no candidate shared framework is enabled.</value>
+    <value>Roll forward on no candidate framework (0=off, 1=roll minor, 2=roll major &amp; minor).</value>
```

Comment:
Could this be 
```
Roll forward framework (0=off, 1=roll minor, 2=roll major &amp; minor).
```
I thought we did minor roll forward even when exact version was present.

Reply:
We roll forward to latest only on patch version. Minor/Major roll forward to the least greater or equal. So if there's equal we will always pick that one (even if roll forward is allowed). So the answer is no, we don't auto-roll forward on minor unless we have to (and are allowed). In short if the exact match is available, we will use it (baring patch roll forward which is a different matter).